core/src/main/resources/hudson/triggers/SCMTrigger/config.jelly

```diff
@@ -25,7 +25,7 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <f:entry title="${%Schedule}" help="/descriptor/hudson.triggers.TimerTrigger/help/spec">
-    <f:textarea field="scmpoll_spec" checkUrl="'${rootURL}/trigger/TimerTrigger/check?value='+encodeURIComponent(this.value)"/>
+    <f:textarea field="scmpoll_spec" checkUrl="'descriptorByName/hudson.triggers.TimerTrigger/checkSpec?value='+encodeURIComponent(this.value)"/>
```

Comment:
@jtnord  This calls http://javadoc.jenkins-ci.org/hudson/model/AbstractItem.html#getDescriptorByName%28java.lang.String%29 and it needs to for the `H` pattern in cron expressions, which are project specific and use `AncestorInPath`:
https://github.com/jenkinsci/jenkins/blob/a07b127a1acf015aafce2961646513eefe5da202/core/src/main/java/hudson/triggers/TimerTrigger.java#L87

Otherwise, no way to resolve this issue of incorrect predictions in the 'success' case.